libminifi/test/KamikazeProcessor.cpp

```diff
@@ -53,14 +53,13 @@ void KamikazeProcessor::initialize() {
 
 void KamikazeProcessor::onSchedule(core::ProcessContext *context, core::ProcessSessionFactory* /*sessionFactory*/) {
   std::string value;
-  bool bool_value;
-
   _throwInOnTrigger = false;
```

Comment:
this line can be deleted, too

Reply:
It's already deleted I guess

Reply:
I meant `_throwInOnTrigger = false;` can be deleted, too

Reply:
oh okay. Sorry I got it wrong the first time.